{"name": "test_check_base_and_config", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dhamyan\\\\PycharmProjects\\\\PythonProject\\\\PythonProject\\\\qa-automation-api\\\\test\\\\data\\\\endpoints_config.json'", "trace": "@pytest.fixture(scope=\"session\")\n    def load_config():\n        config_path = os.path.join(os.path.dirname(__file__), \"data\", \"endpoints_config.json\")\n>       with open(config_path, \"r\") as f:\n             ^^^^^^^^^^^^^^^^^^^^^^\nE       FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\dhamyan\\\\PycharmProjects\\\\PythonProject\\\\PythonProject\\\\qa-automation-api\\\\test\\\\data\\\\endpoints_config.json'\n\nconftest.py:12: FileNotFoundError"}, "start": 1759428482436, "stop": 1759428482436, "uuid": "d61ac780-d480-41bd-ab9f-f42e143a0795", "historyId": "95acc43777b46fe8a61ca0fcd8b50772", "testCaseId": "95acc43777b46fe8a61ca0fcd8b50772", "fullName": "test.test_dummy#test_check_base_and_config", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_dummy"}, {"name": "host", "value": "Jerry"}, {"name": "thread", "value": "2568-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_dummy"}], "titlePath": ["test", "test_dummy.py"]}